scripts/build/gen_isr_tables.py

```diff
@@ -153,7 +153,7 @@ def write_code_irq_vector_table(fp, vt, nv, syms):
     fp.write("}\n")
 
 def write_address_irq_vector_table(fp, vt, nv):
-    fp.write("uintptr_t __irq_vector_table _irq_vector_table[%d] = {\n" % nv)
+    fp.write("uintptr_t __irq_vector_table __attribute__((__used__)) _irq_vector_table[%d] = {\n" % nv)
```

Comment:
You meant `__used` here, right?

Reply:
Are you suggesting changing this to:
```
uintptr_t __irq_vector_table __used _irq_vector_table[%d] =
```
to make it more cross-compiler friendly?

Reply:
yes, to make it toolchain agnostic.